butterfree/pipelines/feature_set_pipeline_metadata.py

```diff
@@ -0,0 +1,220 @@
+from typing import List, Literal, Optional, Union
+
+from pydantic import BaseModel, ConfigDict, Field, field_serializer
+from typing_extensions import Annotated
+
+from butterfree.extract.readers.reader import (
+    FileReaderMetadata,
+    KafkaReaderMetadata,
+    TableReaderMetadata,
+)
+from butterfree.load.writers.writer import WriterMetadata
+from butterfree.pipelines import FeatureSetPipeline
+from butterfree.transform.aggregated_feature_set import AggregatedFeatureSet
+
+
+class Column(BaseModel):
+    """Metadata model for a column in a feature set.
+
+    This model represents the metadata of a single column in a feature set,
+    including its name, data type, and whether it's a primary key.
+    """
+
+    model_config = ConfigDict(arbitrary_types_allowed=True)
```

Comment:
We store the model config in the metadata related to a feature? What if more than one model uses this feature?

Reply:
This is a field for configuration of pydantic models